3scale/porta
# ♻️  Refactors plans_base_presenter

**What this PR does / why we need it**:

Using update_plans_controllers refactor branch, this PR continues refactoring to solve the Default plans / Table plans problems described at THREESCALE-8066 Jira Ticket.

**Which issue(s) this PR fixes** 

[THREESCALE-8066](https://issues.redhat.com/browse/THREESCALE-8066) Default plan card disappears when searching for plans. 

**Verification steps** 

> 1. Go to Application Plans.
> 2. Search for an Application plan.
> 3. Confirm that Default plan card is still available and that the dropdown has information.
